src/components/SectionSwap.tsx

```diff
@@ -0,0 +1,100 @@
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+import { useState } from 'react';
+import { motion } from 'framer-motion';
+import { FiChevronDown } from 'react-icons/fi';
+
+import CustomInput from 'components/CustomInput';
+import DialogSwap from 'components/DialogSwap';
+import { useAtomValue } from 'jotai';
+import { brandToInfoAtom, displayFunctionsAtom } from 'store/app';
+import { displayPetname } from 'utils/displayFunctions';
+
+const SectionSwap = ({
+  type,
+  value,
+  handleChange,
+}: {
+  type: string;
+  value: bigint;
+  handleChange: (value: bigint) => void;
+}) => {
+  const { displayBrandIcon, displayBrandPetname } =
+    useAtomValue(displayFunctionsAtom);
+  const brandToInfo = useAtomValue(brandToInfoAtom);
+
+  const [open, setOpen] = useState(false);
+
+  const handleBrandSelected = () => {
+    console.log('TODO: handle brand selected');
+  };
+
+  const handlePurseSelected = () => {
+    console.log('TODO: handle purse selected');
+  };
+
+  // TODO: Filter brands.
+  const brands = [...brandToInfo.keys()];
+  const brand = undefined;
+  const purse: any | undefined = undefined;
```

Comment:
I missed this in 5f7ea6c . 
```suggestion
  const purse: PursesJSONState | undefined = undefined;
```
btw, when do you choose `undefined` vs `null`?

Reply:
Good question. I think a lot of these should be null, just changed that. I was just giving it undefined to make the static analysis work but these states will be handled for real in the next PR.